### PR TITLE
docs(README): mention nvm windows

### LINKED ISSUE #317: [DOCS] Update ReadMe to better support Windows users
### Describe the feature you would like to request

The ReadMe is very well detailed and easy to follow. 
In the prerequisite section, it does mention nvm except that nvm doesn't provide support for Windows. 

As a Windows user, I want to know what are my options 
So I'm able to build and run the extension locally 

### Describe the solution you would like

Update the ReadMe to mention the nvm for windows project which allows Windows users to achieve the same results as with nvm 
https://github.com/coreybutler/nvm-windows

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by raducristianpopa**
## Context

Closes #317.

### CODE REVIEW

**Comment by raducristianpopa** [comment]
<h2>Extension builds preview</h2>

<table role="table">
<thead>
  <tr>
    <th>Name</th>
    <th>Link</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td align="center">Latest commit</td>
    <td align="center">972ca4de47ceb23b797a149250e2999d8af9c288</td>
  </tr>
  <tr>
    <td align="center">Latest job logs</td>
    <td align="center"><a href="https://github.com/interledger/web-monetization-extension/actions/runs/9834284861">Run #9834284861</a></td>
  </tr>
  <tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/25735480848/artifacts/1676319909">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/25735480848/artifacts/1676319910">Download</a></td></tr>
</tbody>
</table>

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The Web Monetization browser extension is an open source implementation of the W
 
 ### Prerequisites
 
-- [NVM](https://github.com/nvm-sh/nvm) - or another Node Version Manager
+- [NVM](https://github.com/nvm-sh/nvm) (Linux, macOS), [NVM Windows](https://github.com/coreybutler/nvm-windows) (Windows) - or another Node Version Manager
```
Comment (sidvishnoi) [inline README.md:12]: @dianafulga [mentioned](https://interledgerfoundation.slack.com/archives/C06CVJRT5N3/p1720008488651479?thread_ts=1720008408.321259&cid=C06CVJRT5N3) `nvm install lts/iron` does not work on Windows. I checked docs and they suggest "lts" (not "lts/iron") - so could be that?

Comment (raducristianpopa) [reply README.md:12]: `nvm install lts` does not work for me with https://github.com/nvm-sh/nvm.

Comment (raducristianpopa) [reply README.md:12]: https://github.com/interledger/web-monetization-extension/pull/398/commits/e0201d5595bec75b98743061355da82190d84097 This should be better.

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**